config/routes.rb

```diff
@@ -17,6 +17,9 @@
   namespace :admin do
     get '/dashboard', to: "dashboard#index"
     get '/users', to: "dashboard#show_users"
+    get '/merchants', to: "dashboard#show_merchants"
+    get 'merchant/:id/disable', to: "dashboard#disable_merchant"
+    get 'merchant/:id/enable', to: "dashboard#enable_merchant"
```

Comment:
```suggestion
    get '/merchants', to: "dashboard#show"
    get 'merchant/:id/disable', to: "dashboard#disable"
    get 'merchant/:id/enable', to: "dashboard#enable"
```
We'd have to change the actions on the controllers, but can't we remove `_merchant` from the `show`, `disable`, `enable` actions?